extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -536,37 +574,152 @@ export class AmpStoryEmbeddedComponent {
   }
 
   /**
-   * Animates into expanded view.
+   * Returns expanded element back to original state.
+   * @private
+   */
+  closeExpandedEl_() {
+    this.triggeringTarget_.classList.toggle(
+        'i-amphtml-expanded-component', false);
+    const embedId =
+      this.triggeringTarget_.getAttribute(EMBED_ID_ATTRIBUTE_NAME);
+
+    const styleEl = embedStyleEls[embedId];
+
+    const transformString = styleEl.textContent.substring(
+        styleEl.textContent.indexOf('transform'),
+        styleEl.textContent.indexOf('margin'));
+
+    embedStyleEls[embedId].textContent =
+      styleEl.textContent.replace(transformString,
+          `transform: scale(${styleEl[AMP_EMBED_DATA].scale}) !important;`);
+  }
+
+  /**
+   * Animates into expanded view. It calculates what the full-screen dimensions
+   * of the element will be, and uses them to deduce the translateX/Y values
+   * once the element reaches its full-screen size.
    * @param {!Element} target
    * @private
    */
   animateExpanded_(target) {
+    const embedId = target.getAttribute(EMBED_ID_ATTRIBUTE_NAME);
     const state = {};
+    const storedStyle = embedStyleEls[embedId][AMP_EMBED_DATA];
```

Comment:
Nothing specific to this line, but a general naming tip: if the variable contains what you call `EMBED_DATA` throughout the file, it'd help future developers if the variable uses the same wording :)

Reply:
Done.